docs/basic-features/layouts.md

```diff
@@ -102,8 +102,9 @@ When using TypeScript, you must first create a new type for your pages which inc
 import type { ReactElement } from 'react'
 import Layout from '../components/layout'
 import NestedLayout from '../components/nested-layout'
+import type { NextPageWithLayout } from './_app'
 
-export default function Page() {
+export default function Page(): NextPageWithLayout {
```

Comment:
This looks incorrect as this is saying the return type should be a `NextPageWithLayout` not that the Page function itself should be that e.g. shouldn't this be:

```ts
const Page: NextPageWithLayout = () => {
  return <p>hello world</p>
}
export default Page
```

Reply:
You are right. My mistake!